deploy_nixos/nixos-deploy.sh

```diff
@@ -30,15 +30,18 @@ sshOpts=(
 drvPath="$1"
 outPath="$2"
 targetHost="$3"
-buildOnTarget="$4"
-sshPrivateKeyFile="$5"
-action="$6"
-shift 6
+targetPort="$4"
+buildOnTarget="$5"
+sshPrivateKeyFile="$6"
+action="$7"
+shift 7
 
 # remove the last argument
 set -- "${@:1:$(($# - 1))}"
 buildArgs+=("$@")
 
+sshOpts+=( -p "${targetPort}" )
```

Comment:
Can you use the same strategy as `sshPrivateKeyFile` to not set the parameter if the default value (`0` I guess?) is used. Otherwise, it might override the `~/.ssh/config`.

Reply:
i opted for explicitly defining the default-value for the variable and use that through out.

my reasoning is: the terraform-provisioner-connection defaults to port 22 for ssh when not explicitly provided: https://www.terraform.io/docs/provisioners/connection.html#port

Reply:
Alright, this is something can can be changed back in either case.

Reply:
thanks for merging :)